include/seahorn/Transforms/Instrumentation/GeneratePartialFnPass.hh

```diff
@@ -0,0 +1,49 @@
+
+#ifndef __GENERATE_PARTIAL_FN_PASS__HH_
+#define __GENERATE_PARTIAL_FN_PASS__HH_
+
+/**
+ * Locates external function calls annotated by "partial" and generates
+ * placeholder implementations.
+ */
+
+#include "llvm/IR/Function.h"
+#include "llvm/Pass.h"
+
+#include <set>
+#include <string>
+
+using namespace llvm;
+
+namespace seahorn {
+
+class SeaBuiltinsInfo;
+
+class GeneratePartialFnPass : public llvm::ModulePass {
```

Comment:
this pass does not expose any public api. it does not need to be exposed in a header file